docs/blog/posts/2022-10-10-work_with_large_zip_files_in_python.md

```diff
@@ -0,0 +1,136 @@
+---
+date:
+  created: 2022-10-10
+  updated: 2024-11-19
+categories:
+    - zip
+authors:
+    - kevin128
+subtitle: Working with Large Zip Files in Python
+---
+
+# Working with Large Zip Files in Python
+
+## How to work with large zip files **without** unzipping them
+
+### Problem
+You need to access data from a zip file, but you don’t want to copy the zip file to your home/project directory and unzip it. How can you access this data efficiently?
+
```

Comment:
We can add the `more` separator so that the blog is previewed only to that point

Reply:
this is what we add to all blog pages so that only the preview part before the more is shown

Reply:
see this: https://github.com/gsbdarc/rcpedia_mockup/blob/QA/instructions/making_blogs.md